app/client/templates/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,182 @@
+import { Layout } from "@client/components/Layout";
+import React from "react";
+
+export const Template = () => {
```

Comment:
This is quite ugly. I would try to make it look better, maybe by using `react-markdown`? Then it will be easier to edit.

Reply:
Done